project/cmake/CMakeLists.txt

```diff
@@ -46,16 +46,16 @@ option(ENABLE_AIRTUNES    "Enable AirTunes support?" ON)
 option(ENABLE_CEC         "Enable CEC support?" ON)
 option(ENABLE_OPTICAL     "Enable optical support?" ON)
 if(UNIX)
-  if(NOT APPLE)
-    option(ENABLE_INTERNAL_FFMPEG "Enable internal ffmpeg?" ON)
-  endif()
   option(FFMPEG_PATH        "Path to external ffmpeg?" "")
   option(ENABLE_INTERNAL_CROSSGUID "Enable internal crossguid?" ON)
   option(ENABLE_OPENSSL     "Enable OpenSSL?" ON)
   option(ENABLE_SDL         "Enable SDL?" OFF)
   if(CORE_SYSTEM_NAME STREQUAL linux OR CORE_SYSTEM_NAME STREQUAL freebsd)
     option(ENABLE_X11         "Enable X11 support?" ON)
     option(ENABLE_AML         "Enable AML?" OFF)
+    option(ENABLE_INTERNAL_FFMPEG "Enable internal ffmpeg?" OFF)
```

Comment:
The module is quite difficult to read, maybe it's a good idea to have a comment in the module that explains the default behavior. Assuming I build on linux without depends installed, it would now try to find ffmpeg 3.1, and because I don't have that installed, it would build the kodi ffmpeg version for me, correct?